docs/tutorials/example_case_files/a_example_stavanger_with_commands.yaml

```
@@ -0,0 +1,84 @@
+time:
+  starttime: 2024-05-31 22:00:00 # UTC time stamped, for Oslo this means midnight June 1st.
+  endtime: 2024-06-02 22:00:00 # Typically set to a later time than 24 hrs such that the optimization accounts for next day too.
```

```suggestion
  endtime: 2024-06-02 22:00:00 # Even if we mostly care about the first 24 hours, it can be wise to optimise for a longer period.
```
Something like this, or just remove the comment? (The typical set-up is 1-3 weeks ahead, not just 2 days, but I don't think it makes sense to mention/explain/discuss that in detail here.)